internal/exportimport/model/file.go

```diff
@@ -30,4 +31,24 @@ type ImportFile struct {
 	DiscoveredAt   time.Time
 	ProcessedAt    *time.Time
 	Status         string
+	Retries        uint
+}
+
+// ShouldTry performs some introspection on an import file from the DB, and
+// returns true if that file should be tried for download.
+func (f *ImportFile) ShouldTry() bool {
+	if f.Status == ImportFileRetry {
+		// ProcessedAt should be non-null for all codepaths that set status to retry.
+		d := *f.ProcessedAt
```

Comment:
i would like to see the retry parameters configurable and seems like all retry units should be just a time function

Reply:
This should do it fer ya!